<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewFittingWindow</class>
 <widget class="QMainWindow" name="NewFittingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fitting Result</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>560</width>
      <height>150</height>
     </rect>
    </property>
    <property name="title">
     <string>Fitting for New Gene Profile Curve</string>
    </property>
    <widget class="QLabel" name="labelNewModelADescription">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>75</y>
       <width>140</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelNewModelA_Parameters">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>300</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelNewModelA_AValue95Interval">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>65</y>
       <width>300</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelNewModelA_BValue95Interval">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>90</y>
       <width>300</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelNewModelA_RSquared">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>115</y>
       <width>300</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelNewModelA">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>35</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelNModelName1_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>200</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Parameters and Fitting Result:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelNModelName1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>120</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Model:</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
